oh-my-bash.sh

```diff
@@ -144,6 +144,12 @@ unset -v _omb_init_files _omb_init_file
 # Load the theme
 if [[ $OSH_THEME == random ]]; then
   _omb_util_glob_expand _omb_init_files '"$OSH"/themes/*/*.theme.sh'
+
+  # Remove ignored themes from the list
+  for theme in random ${OMB_THEME_RANDOM_IGNORED[@]}; do
```

Comment:
- The array expansion needs to be properly quoted in case `IFS` is set to a custom value.
- Also, the variable name `theme` can conflict with a user-defined variable. The variable name should be prefixed by `_omb_init_`. At the end of the block, the variable should be unset by `unset -v`.


```suggestion
  for _omb_init_theme in random "${OMB_THEME_RANDOM_IGNORED[@]}"; do
```